content/_index.md

```diff
@@ -79,6 +80,13 @@ These crates are under active development.
 
  - [`linebender.github.io`][linebender.github.io] - This website. If you see anything that looks incorrect, please submit a PR to [the repo][linebender.github.io].
 
+## Archived projects
+
+ - [`glazier`][glazier] - A windowing library.
+
+  Glazier was created due to limitations with [`winit`][winit], however we have since decided to work together with the winit team to resolve those issues.
+  We now recommend the use of winit over Glazier.
```

Comment:
```suggestion
  Glazier was created due to limitations with [`winit`][winit], however we have since decided to work together with the `winit` team to resolve those issues.
  We now recommend the use of `winit` over Glazier.
```